minari/dataset/minari_dataset.py

```diff
@@ -213,7 +207,7 @@ def filter_episodes(
         ```
 
         Args:
-            condition (Callable[[Any], bool]): callable that accepts any type(For our current backend, an h5py episode group) and returns True if certain condition is met.
+            condition (Callable[[EpisodeData], bool]): callable that accepts any type(For our current backend, an h5py episode group) and returns True if certain condition is met.
```

Comment:
Update `any type` with EpisodeData and remove parenthesis `(For our current backend, an h5py episode group)`